Templates/Common-Backend-Scripts/README.md

```diff
@@ -802,9 +802,9 @@ CLI parameter | Description
 -o `<packageOutputFile>` | (Optional) Absolute or relative path to the **Package Output File** that specifies the location where to store the downloaded tar file. If providing a relative path, the file path is prefixed with Wazi Deploy Packaging directory `<wdDeployPackageDir>` configured in `pipelineBackend.config`. Only required when wazideploy-generate is used to download the package. This is indicated when a URL is specified for the **Package Input File**.
 -d | (Optional) Debug tracing flag. Used to produce additional tracing with Wazi Deploy.
 -- | - when retrieving the tar file from Artifact repo the below options are mandatory -
--b `<branch>`| Name of the **git branch** turning into a segment of the directory path for the location within the artifact repository.
+-b `<branch>`| (Deprecated) Name of the **git branch** turning into a segment of the directory path for the location within the artifact repository for the `build` pipelineType.
 -p `<build/release>` | **Pipeline Type** to indicate a `build` pipeline (build only with test/debug options) or a `release` pipeline (build for optimized load modules for release candidates).
--R `<releaseIdentifier>` | **Release identifier** to indicate the next planned release name. This is a computed value based on the pipeline templates. It can be automatically computed using the [computeReleaseVersion.sh](#computereleaseversionsh) script.
+-R `<releaseIdentifier>` | **Package identifier** to indicate the origin of the package. In case of a Release candidate, the release identifier such as `rel-1.0.0`. This is usually a computed value based on the pipeline templates. It can be automatically computed using the [computeReleaseVersion.sh](#computereleaseversionsh) script. In case of a reliminary build (pipelineType `build`), the value for releaseIdentifier is the branch name of which the package originates from.
```

Comment:
```suggestion
-R `<packageIdentifier>` | **Package identifier** to indicate the origin of the package. In case of a Release candidate, the release identifier is such as `rel-1.0.0`, with a `rel-`prefix followed by a version number. This is usually a computed value based on the pipeline templates. It can be automatically computed using the [computeReleaseVersion.sh](#computereleaseversionsh) script. In case of a preliminary build (pipelineType `build`), the value for packageIdentifier is the branch name used for building artifacts stored in the package.
```